fastai/tabular/data.py

```diff
@@ -27,7 +27,7 @@ def from_csv(cls, csv, skipinitialspace=True, **kwargs):
         return cls.from_df(pd.read_csv(csv, skipinitialspace=skipinitialspace), **kwargs)
 
     @delegates(TabDataLoader.__init__)
-    def test_dl(self, test_items, rm_type_tfms=None, process=True, **kwargs):
```

Comment:
Why is this removed?

Reply:
We don't ever pass in `rm_type_tfms` for anything and it doesn't serve a purpose for tabular. (Unless you can think of a scenario we can keep it).